app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@
         android:theme="@style/AppTheme">
         <activity android:name=".EducationPage"></activity>
         <activity android:name=".openDayDetails"></activity>
-        <activity android:name=".MainActivity">
```

Comment:
For purposes of testing, this will be temporarily changed to "QuestionFormActivity" until the commits have been approved. After that, and BEFORE MERGING, the code will be edited to say MainActivity again.